client/my-sites/domains/domain-management/dns/dns-add-new.jsx

```diff
@@ -204,7 +204,7 @@ class DnsAddNew extends React.Component {
 				{ this.recordFields() }
 				<FormFooter>
 					<FormButton disabled={ isSubmitDisabled } onClick={ this.onAddDnsRecord }>
-						{ translate( 'Add New DNS Record' ) }
+						{ translate( 'Add new DNS record' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Add new DNS records' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/button-sentence-case)